src/code/z_collision_check.c

```diff
@@ -2695,7 +2693,7 @@ typedef enum {
     /* 2 */ MASSTYPE_NORMAL
 } ColChkMassType;
 
-#pragma increment_block_number 253
+#pragma increment_block_number "gc-eu:252 gc-eu-mq:252"
```

Comment:
If I understand things correctly, `increment_block_number` should always be 253 in this particular case because the intent is to increment the block number such that including the definition for `ColChkMassType` here doesn't result in a shift in block number

Reply:
Yeah but there is some wiggle room here and we don't know exactly how it was originally. Including `ColChkMassType` here was just one solution that I found when trying to match BSS in this file originally, but apparently it wasn't popular. We could also shift things slightly by playing with variable scoping, for example